src/periphery/interfaces/ISablierMerkleFactory.sol

```diff
@@ -68,12 +84,30 @@ interface ISablierMerkleFactory {
         pure
         returns (bool result);
 
+    /// @notice Retrieves the default sablier fee required to claim an airstream.
+    /// @dev A minimum of this fee must be paid in ETH during `claim`.
+    function defaultSablierFee() external view returns (uint256);
+
+    /// @notice Retrieves the custom sablier fee struct for a specified campaign creator.
+    /// @dev It return two fields:
```

Comment:
let's remove this `dev` natspec as the struct is already explained in `DataTypes`

Reply:
Fair point. I wanted to add a `@return` tag but couldn't figure out a way to I decided to include the `@dev` tag.